flux-kube/configure.ac

```diff
@@ -52,6 +52,9 @@ AC_SUBST(fluxcmddir)
 AS_VAR_SET(jobtap_plugindir, $libdir/flux/job-manager/plugins)
 AC_SUBST(jobtap_plugindir)
 
+AS_VAR_SET(fluxrc1dir, $sysconfdir/flux/rc1.d)
+AC_SUBST(fluxrc1dir)
+
```

Comment:
Just out of curiosity, you don't need a matching `rc3` script?

Reply:
No, it works fine with just rc1. I don't understand the `rc`s in this context very well, so why is that surprising?